Added mjh5810 post and accompanying images

mjh5810 opened the pull request.

Bestest-Coder committed:
Added mjh5810 post and accompanying images

Bestest-Coder committed:
Added post title

MoralCode commented on _posts/2025-04-06-mjh5810-nixos-frp.md:
```diff
@@ -0,0 +1,28 @@
+
+title: "Improving Documentation in NixOS Modules"
+
+----
```
you have an extra dash here thats breaking stuff

MoralCode replied:
![Screenshot_20250406_212254](https://github.com/user-attachments/assets/4cb4d89c-f60a-4d83-a55d-079c508930c5)

MoralCode reviewed (changes requested):
general feedback: looks good! images are working okay

i would recommend making sure you address all the sub-points for each of the required sections in the outline (and ideally take a look at the rubric ill be using to grade things - which is attached to the assignment on MyCourses).

let me know when you think this is ready to merge (so that you have time to make any tweaks beyond the functionality fixes

Bestest-Coder committed:
fixed metadata extra dash and added some more detail

mjh5810 commented:
Fixed the metadata issue and added a little more detail, although I don't have too much to say considering it was a pretty small change that hasn't been accepted yet. I was thinking of doing something more complex, which I will probably still do after this assignment, but I ended up busy with other things.
